.travis.yml

```diff
@@ -47,7 +47,6 @@ install:
   - firefox --version
   # lint python
   - pep8 --ignore=E402 webcompat/ tests/ config/secrets.py.example
```

Comment:
This seems like a sensible change to me. 👍  
Does it make any sense to move the `pep8` call into the `before_script` section, while we're at it (for consistency with other linting)?

Reply:
That sounds reasonable.